nice file.txt

```diff
@@ -0,0 +1 @@
+very nice
```

Comment:
not very good line

Reply:
no no! you are wrong. this is a great line!